ros2param/ros2param/verb/dump.py

```diff
@@ -64,7 +69,7 @@ def get_parameter_value(node, node_name, param):
 
         # requested parameter not set
         if not response.values:
-            return '# Parameter not set'
+            return None
```

Comment:
okay, humble does not have this issue (does not use `zip` method), it returns '# Parameter not set' string and set it into the dictionary with parameter name. it shows, `parameter_name: # Parameter not set`.

besides, we cannot backport https://github.com/ros2/rclpy/pull/1216 to humble because it breaks the user space, that means this return '# Parameter not set' situation would happen even with statically typing parameter without default value. instead of modifying and changing behavior to handle this '# Parameter not set' case with exception, probably it is better to keep the current behavior as humble.